### DISCUSSION TITLE
Mintpy doesn't run

### DISCUSSION THREAD

**Discussion by MahdRaz** [alexisInSAR/EZ-InSAR#70] [Help]
Hi there,
I am trying to run the third module, both are having problems on the "RUN" and "RUN ALL STEPS" stage, here are the Mintpy error:

Error using textscan
Invalid file identifier. Use fopen to generate a valid file identifier. 

and here is for step by step errors:
step - modify_network and step - reference_point and step - deramp: FileNotFoundError: [Errno 2] No such file or directory: '/home/.../stackmintpy/inputs/ifgramStack.h5' and 'stackmintpy/timeseries.h5' and 'stackmintpy/velocity.h5'.

I'm guessing that both Mintpy and StaMPS are not working correctly on my computer.

Thank you for your help!

**Comment by alexisInSAR** [comment]
Hi, 

Did your ISCE processing work correctly? 

Kind regards, 

Alexis

**Reply by MahdRaz** [reply]
Seemed okay except when I went for "select the best reference date", I got this:

Error using imread>get_full_filename
File "/home.../coarse_ifg_network.jpg" does not exist.

Thanks, Alexis

**Reply by MahdRaz** [reply]
a bit of more explanation:

 File "/home ... envs/InSARenv/lib/python3.11/site-packages/numpy/lib/nanfunctions.py", line 552, in nanargmin
    raise ValueError("All-NaN slice encountered")
ValueError: All-NaN slice encountered
Error using imread>get_full_filename
File "/home/ ... /coarse_ifg_network.jpg" does not exist.

**Comment by MahdRaz** [comment]
Hi Alexis,

I was not having any issues until Module 3, but when I start it from zero again to see if everything is going okay, I got this error:
![Screenshot from 2023-11-22 20-34-25](https://github.com/alexisInSAR/EZ-InSAR/assets/32454623/5e53e0d4-0a3d-4d2e-8e32-77bb537df46a)

I can see that this error happened for some before and it was because of cv2, but mine is different which I don't get it why. I tried over and over and every time I was stopped with this error. Any idea?

Thanks for your help!

**Reply by alexisInSAR** [reply]
Hi, 

Please use the version of EZ-InSAR (2.2.0) and let me know if it works. 

Kind regards, 

Alexis

**Reply by MahdRaz** [reply]
Hi Alexis and happy new year!
I tried with version 2.2.0 and still have the same error. I would appreciate you help.
Best,
M
